stages/org.osbuild.kickstart

```diff
@@ -438,6 +477,29 @@ def make_users(users: Dict) -> List[str]:
     return res
 
 
+def make_rootpw(options: Dict) -> str:
+    rootpw = options.get("rootpw")
+    if rootpw is None:
+        return ""
+    cmd = "rootpw"
+    plaintext = rootpw.get("plaintext", False)
```

Comment:
I don't think there's really any need for intermediate variable names in these if checks. But that's a style thing.

Reply:
related change has been done in this commit https://github.com/osbuild/osbuild/pull/1560/commits/903b5622b703aa8d9a77acc4ce8a814cf2dd877c